.github/workflows/healthchecks_elasticsearch_ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         image: docker.elastic.co/elasticsearch/elasticsearch:6.3.2
         ports:
           - 9300:9300
-          - 9200:9200
+          - 9201:9200

```

Comment:
?

Reply:
To answer your question, there was a "hybrid" approach to integration/functional tests:
- some tests used the Docker container created via CI pipeline
- some tests created a new Docker container (via .NET SDK)

And both of them tried to run the container on port 9200, so there was a conflict.

I changed the Docker setup for CI pipeline to use different port - 9201, because I knew how to do that, as opposed to the second one :D.

And now all of the integration/functional tests are passing!